tests/ops/functions/test_iterative_qpe.py

```diff
@@ -237,3 +237,51 @@ def circuit_iterative():
             return [qml.expval(op=i) for i in measurements]
 
         assert np.allclose(circuit_qpe(), circuit_iterative())
+
+
+@pytest.mark.slow
```

Comment:
My laptop takes around 1 minute and 20 seconds to run this, so I agree to mark it as slow

Reply:
didn't actually know it was that bad 😢 . Really need to figure out a better way to test this.